Rakefile

```diff
@@ -444,13 +457,14 @@ which are simply available in the repository.
     port = args.to_hash.fetch(:port, 5000)
     require 'mkmf'
     autobuild_cmd = find_executable 'sphinx-autobuild'
-    if autobuild_cmd && false
-      cmd = "SIMP_FAST_DOCS=true #{autobuild_cmd} -p #{port} -H 0.0.0.0 --poll --ignore docs/dynamic/\*.rst docs html"
-      run_build_cmd(cmd)
+    env_str="SIMP_FAST_DOCS=true"
+    if autobuild_cmd
+      cmd = "#{env_str} #{autobuild_cmd} -p #{port} -H 0.0.0.0 --poll --ignore docs/dynamic/\*.rst docs html"
+      run_streaming_cmd(cmd)
     else
       puts "running web server on http://localhost:#{port}"
       Rake::Task['docs:html'].invoke
```

Comment:
This task (docs:html) does not run in autobuild_cmd returns nil.  So I don't see any point in invoking it.

Reply:
Ignore my previous comment,  docs:html doesn't work iv it can't find sphinx-build, not sphinx-autobuild